### PR TITLE
Feature/show edition name flag

### PR DESCRIPTION
**Pull request by jondewijones**
### What

Check edition title and set ShowEditionName value

### Who can review

Anyone

### CODE REVIEW

**Comment by dandwal** [comment]
Approved as this feedback is not blocking.

I think it might be worth making an array of terms that instead of checking just if the edition is exactly 'time-series'. 
Reasons:
* Right now it is acting a bit like a magic string, variable, editionNamesNotToShowSomePage although long implicitly explains what this check is for. Or use a shorter variable name and add a comment above it, this would mean it is no longer a magic string dictating functionality without explanation.
* Right now this is arguably closed to expansion, as if you want to add more types of editions that don't show the list editions page then we end up creating a longer and longer selection statement, adding more and more conditions. Condition statements should be short and clear as unlike other areas of the program the logic isn't linear; thus harder to maintain.

In summary, with an array of terms which just has ['time-series'] in, for now, we can just append then array with new terms if we expand this functionality. The selection statement never needs to be touched again. (until we put in a better solution for handling this.) It adds a layer of clarity to what is going on and why.
